### PR TITLE
Fix comment of `SuspendTime`

### PR DESCRIPTION
**Pull request by ekexium**
Based on current TiKV implementation, modify the comments to make them more precise.

### CODE REVIEW

File: util/execdetails.go [hunk 1]
```diff
@@ -609,12 +609,13 @@ type TimeDetail struct {
...
 	// this field is called wall time instead of CPU time.
 	ProcessTime time.Duration
-	// Cpu wall time elapsed that task is waiting in queue.
+	// Time elapsed when a coprocessor task yields itself.
 	SuspendTime time.Duration
 	// Off-cpu wall time elapsed in TiKV side. Usually this includes queue waiting time and
 	// other kind of waits in series.
 	WaitTime time.Duration
-	// KvReadWallTime is the time used in KV Scan/Get.
+	// KvReadWallTime is the time used in KV Scan/Get. For get/batch_get,
```
Comment (glorv) [inline util/execdetails.go:617]:
```suggestion
	// KvReadWallTime is the time used in KV Scan/Get. For coprocessor requests,
```
Coprocessor request queue time is more notable due to reschedule

Comment (ekexium) [reply util/execdetails.go:617]: My point was that for get/batch_get, the field covers more procedures

Comment (glorv) [reply util/execdetails.go:617]: oo, you are right.

**Review by glorv — APPROVED** [review]
LGTM

**Review by disksing — APPROVED** [review]

**Pull request merged**